commons/c2cgeoportal_commons/models/main.py

```diff
@@ -1189,12 +1222,43 @@ class LayerVectorTiles(DimensionLayer):
         info={
             "colanderalchemy": {
                 "title": _("Raster URL"),
-                "description": "The raster url. Example: https://url/{z}/{x}/{y}.png",
+                "description": _(
+                    """
+                    The raster url. Example: https://url/{z}/{x}/{y}.png. Alternative to the SQL query field.
+                    """
+                ),
                 "column": 2,
             }
         },
     )
 
+    sql = Column(
+        Unicode,
+        nullable=True,
+        info={
+            "colanderalchemy": {
+                "title": _("SQL query"),
+                "description": _(
+                    """
+                    A SQL query to get the vector tiles data. Alternative to the Raster URL field.
```

Comment:
```suggestion
                    A SQL query to get the vector tiles data.
```

Reply:
Is this for exposing an MVT endpoint using directly ST_AsMVT?

Reply:
Maybe it should be more clear how this SQL should look like. Or at least put a pointer to some external doc about it.

Reply:
@gberaudo yes, we will clarify this when we will write the service :-)